### PR TITLE
Code Viewer Component

### PR DESCRIPTION
**Pull request by lichliter**

### CODE REVIEW

**Comment by lichliter** [comment]
CleanShot 2023-03-22 at 16 38 21

File: code-viewer/components/codeViewer/index.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      name: 'language',
+      type: PropType.String,
+      value: 'javascript',
+      description: 'The language of the code.',
+    },
+  ],
+  render({props, className, ref, id }) {
+    return (
+      <span data-uiflow-component-id={id} ref={ref} className={className}>
+        <SyntaxHighlighter language={props.language} style={okaidia} wrapLongLines className={className} >
```
Comment (davidUiflow) [inline code-viewer/components/codeViewer/index.tsx:33]: can you also expose `showLineNumbers` and `wrapLongLines`?

**Review by davidUiflow — APPROVED** [review]

File: code-viewer/components/codeViewer/index.tsx (hunk 1 above)
Comment (jimmyhchan) [inline code-viewer/components/codeViewer/index.tsx:33]: looking at the docs it seems the className is used if `useInlineStyles` is false. Do we need to have className here?

Comment (lichliter) [reply code-viewer/components/codeViewer/index.tsx]: idk man its hard. Something about me needing to apply ref to a parent container and hacking the styles to work in this case.

**Pull request merged**